### ISSUE TITLE
[1.20.1] Possible Backup Corruption

### ISSUE THREAD

**Issue by Epik-B0mber** [ChaoticTrials/SimpleBackups#43]
### Minecraft version

1.20.x

### Simple Backups version

1.20.1-3.1.8

### (Neo)Forge version

47.3.0

### The latest.log file

n/a

### Issue description

Hello! I am unsure if this is actually an issue with this mod or not, but wanted to put out an issue about it just in-case, since the probability of outside tampering to me seems... I just don't know how it would be plausible.
Anyways, long story short, I have a server with a terrible disposition to crashing right now as I feverently try to learn how to be a better modder, lol, and have to rely on simplebackups way often than not to get content back, as well as corrupted chunks.
One thing I just noticed today however, being that this is a second-time occasion, is that two chunks in the center of my r.0.0.mca region file keep getting corrupted and purged into normal chunks, something that I luckily was able to revert locally the first time, but, this time, am finding this sudden change to be going back **days** in the backups, which I know for certain is not correct, as I had visited the location last night, and know for sure the corruption was not there.

I have absolutely no idea what may be causing it except my rampant server crashes and corruption, but I'm astonished that a corruption one one day could either somehow dirty days of backups, or that somehow the backup system has been messed up to make all backups do this for some odd reason. If you need any more info I'll try my best but I'm pretty in the dark here myself lol. Thanks!!!

### Steps to reproduce

1. Use backups, seemingly fine so far
2. Get hit with corruption, go to use backup to uncorrupt
3. Find that the corruption somehow spread or was already present in all the copies prior to the corruption
4. Cry

**Comment by MelanX** [comment]
I'm sorry to hear that! 
I can say that the mod doesn't touch any existing backups by using the default config. The only thing is for incremental backups that it looks at the file creation time to check if it should backup this file again or not. 
It doesn't modify existing files, it only copies the current world folder files into a zip file. 

I can't help you with your problem, I have no idea what's the cause for this. Maybe you just got too many corrupted backups because the time between two backups is too low. If you have any additional information which would show me that Simple Backups is wrong, just reopen the issue.

**Comment by Epik-B0mber** [comment]
Hello! Sorry for the possible riff-raff, I would like to inform you it was not your mod it turned out being something else.
Just to share, the idea that the corrupted backups went back made no sense, as, for example I walked into a building that was fully constructed the night before, and then in the morning it had been corrupted; then the backups before I walked into the building were corrupted, and so were a week back, which was impossible-- then finally, I check a local save I _knew_ was fine, and THAT had the same exact corruption too! I thought I was under attack!

Nope, turned out a single mod that I had just updated was freaking out on those two chunks for some odd reason, so it has nothing to do with the backups.
Thanks for the response and good luck with future development!
